### PR TITLE
Create automatically_reassume.md

### PR DESCRIPTION
**Pull request by jordiup**

### CODE REVIEW

**Review by meyerjrr — APPROVED** [review]

File: docs/granted/recipes/automatically_reassume.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Automatically reassume roles 
+
+By default Granted will assume a given role for 1hr. This can be altered by passing the `--duration` flag. This means that after a given hour of activity, you will need to re-run `assume` to get a new set of credentials.
```
Comment (chrnorm) [inline docs/granted/recipes/automatically_reassume.md:3]: @jordiup we also support `duration_seconds` in an AWS profile - https://docs.aws.amazon.com/cli/latest/topic/config-vars.html, can you add a mention of this here?

File: docs/granted/recipes/automatically_reassume.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+If you would like to **automatically reassume** roles, with .zsh you can you can add the following to your `~/.zshrc`:
```
Comment (chrnorm) [inline docs/granted/recipes/automatically_reassume.md:5]:
```suggestion
If you would like to **automatically reassume** roles, with ZSH you can you can add the following to your `~/.zshrc`:
```

File: docs/granted/recipes/automatically_reassume.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Automatically reassume roles
```
Comment (chrnorm) [inline docs/granted/recipes/automatically_reassume.md:1]:
```suggestion
# Automatically reassume roles (ZSH)
```

**Review by chrnorm — CHANGES_REQUESTED** [review]
looks good some minor requested changes

File: docs/granted/recipes/automatically_reassume.md (hunk 1 above)
Comment (jordiup) [reply docs/granted/recipes/automatically_reassume.md]: Cheers Chris, I've added in a mention

**Review by chrnorm — APPROVED** [review]

**Pull request merged**